src/ui/FormElement.tsx

```diff
@@ -36,6 +36,7 @@ const MaxLengthCounter = ({
       fontSize="0.9rem"
       className="text-right"
       maxWidth="43rem"
+      color={currentLength >= maxLength ? 'red' : 'inherit'}
```

Comment:
What in the case of maxLength undefined?

Reply:
This should be good now since this component never renders if we have no maxLength but I can build the security in the component itself worst case